Mergin/validation.py

```diff
@@ -18,6 +19,8 @@
     QGIS_NET_PROVIDERS
 )
 
+INVALID_CHARS = re.compile("[\\\/\(\)\[\]\{\}\"\n\r]")
```

Comment:
@wonder-sk which invlid chars would you include? Wondernig if someone cannot use ( ) in field as description ? is it valid?